scripts/classify_directory.py

```diff
@@ -332,8 +332,11 @@ def make_no_preprocess_config(input_path, output_path):
     with open(input_path) as f:
         config_yaml = yaml.safe_load(f)
         delete_from_dict(config_yaml, 'preprocess')
+        config_yaml['dataset']['images']['preprocess'] = '~'
+        text = yaml.dump(config_yaml)
+        text = text.replace("'~'", "~") # Remove unwanted quotes
```

Comment:
I am worried this might replace other things as well. I think what you want to do is set the preprocess to None earlier?

Reply:
The original version seemed fine in testing but it looks like the result is the same using None.